app/src/main/res/values/strings.xml

```diff
@@ -190,7 +190,13 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="same_request_already_sent">You have already sent a similar request to\u0020</string>
     <string name="relationpage_btntext">Find members</string>
     <string name="refresh">Refresh</string>
-    <string name="error_checkbox">Please choose at least one of the following options.\n You may change it later</string>
```

Comment:
These 2 lines were supposed to stay, can you put them back here. This is from a PR which was merged before yours. TravisCI is failing probably because of this removal

Reply:
are you mentioning the colored lines?

Reply:
yes, the 2 read lines, indicates removal of code. So that is what I am referring to :)

Reply:
done